### ISSUE TITLE
PicoPluginExtensionInitializationException

### ISSUE THREAD

**Issue by G-Rath** [reasonml-editor/reasonml-idea-plugin#163]
plugin version: `0.75`

### Description

Since updating to `0.75`, I get this exception typically once, after I've opened a project from a cold start.

The projects I'm working with right now are mainly TypeScript/Javascript.

```
com.intellij.openapi.extensions.impl.PicoPluginExtensionInitializationException: java.lang.ClassNotFoundException: com.reason.build.annotations.FileHighlighterFilter PluginClassLoader[reasonml, 0.75] com.intellij.ide.plugins.cl.PluginClassLoader@ea9b7a0
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.createInstance(ExtensionComponentAdapter.java:52)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.java:46)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:343)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensions(ExtensionPointImpl.java:282)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensions(ExtensionPointName.java:59)
	at com.intellij.codeInsight.problems.WolfTheProblemSolverImpl.isToBeHighlighted(WolfTheProblemSolverImpl.java:370)
	at com.intellij.codeInsight.problems.WolfTheProblemSolverImpl.reportProblems(WolfTheProblemSolverImpl.java:453)
	at com.intellij.lang.javascript.service.highlighting.JSLanguageServiceAnnotationProvider.apply(JSLanguageServiceAnnotationProvider.java:96)
	at com.intellij.lang.javascript.service.highlighting.JSLanguageServiceHighlightingPass.doApplyInformationToEditor(JSLanguageServiceHighlightingPass.java:133)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.applyInformationToEditor(TextEditorHighlightingPass.java:114)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService.lambda$applyInformationToEditorsLater$3(PassExecutorService.java:499)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:315)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:435)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:419)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:403)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:747)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:696)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.RuntimeException: java.lang.ClassNotFoundException: com.reason.build.annotations.FileHighlighterFilter PluginClassLoader[reasonml, 0.75] com.intellij.ide.plugins.cl.PluginClassLoader@ea9b7a0
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getImplementationClass(ExtensionComponentAdapter.java:96)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.createInstance(ExtensionComponentAdapter.java:40)
	... 31 more
Caused by: java.lang.ClassNotFoundException: com.reason.build.annotations.FileHighlighterFilter PluginClassLoader[reasonml, 0.75] com.intellij.ide.plugins.cl.PluginClassLoader@ea9b7a0
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:51)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getImplementationClass(ExtensionComponentAdapter.java:93)
	... 32 more
Caused by:

java.lang.RuntimeException: java.lang.ClassNotFoundException: com.reason.build.annotations.FileHighlighterFilter PluginClassLoader[reasonml, 0.75] com.intellij.ide.plugins.cl.PluginClassLoader@ea9b7a0
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getImplementationClass(ExtensionComponentAdapter.java:96)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.createInstance(ExtensionComponentAdapter.java:40)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.java:46)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:343)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensions(ExtensionPointImpl.java:282)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensions(ExtensionPointName.java:59)
	at com.intellij.codeInsight.problems.WolfTheProblemSolverImpl.isToBeHighlighted(WolfTheProblemSolverImpl.java:370)
	at com.intellij.codeInsight.problems.WolfTheProblemSolverImpl.reportProblems(WolfTheProblemSolverImpl.java:453)
	at com.intellij.lang.javascript.service.highlighting.JSLanguageServiceAnnotationProvider.apply(JSLanguageServiceAnnotationProvider.java:96)
	at com.intellij.lang.javascript.service.highlighting.JSLanguageServiceHighlightingPass.doApplyInformationToEditor(JSLanguageServiceHighlightingPass.java:133)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.applyInformationToEditor(TextEditorHighlightingPass.java:114)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService.lambda$applyInformationToEditorsLater$3(PassExecutorService.java:499)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:315)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:435)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:419)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:403)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:747)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:696)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.ClassNotFoundException: com.reason.build.annotations.FileHighlighterFilter PluginClassLoader[reasonml, 0.75] com.intellij.ide.plugins.cl.PluginClassLoader@ea9b7a0
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:51)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getImplementationClass(ExtensionComponentAdapter.java:93)
	... 32 more
```

**Comment by giraud** [comment]
It's my fault, I lost a file and only realized it after making the new release. It's annoying but won't impact your work.
It'll be fixed for next release

**Comment by G-Rath** [comment]
> It's annoying

Yes. Yes it is.

Glad to know theres a fix - looking forward to it :joy:

**Comment by giraud** [comment]
fixed in 0.75.1
